test/arrays/ferguson/array-of-owned/array-of-owned-error1.good

```diff
@@ -1,3 +1,4 @@
 array-of-owned-error1.chpl:3: In function 'id':
 array-of-owned-error1.chpl:4: error: invalid copy-initialization
 array-of-owned-error1.chpl:4: error: Cannot copy-initialize array because element type 'owned C' cannot be copy-initialized
+  array-of-owned-error1.chpl:10: called as id(arg: [domain(1,int(64),false)] owned C)
```

Comment:
Is this the first place where we start to indent full lines in error messages? I am not sure what I think about it yet :)

Reply:
The indenting of full lines originated in PR #16175 which you reviewed :)

It is not new in this PR. Either way I'm open to other ways of formatting it.